crypto/dilithium/pqcrystals_dilithium_ref_common/ntt.h

```diff
@@ -4,10 +4,8 @@
 #include <stdint.h>
 #include "params.h"
 
-#define ntt DILITHIUM_NAMESPACE(ntt)
 void ntt(int32_t a[N]);
```

Comment:
These symbol names are probably gonna collide with ML-KEM implementation symbol names if you add this to `bcm.c`.